src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -132,20 +132,29 @@ export const ReviewRoute: FC<ReviewProps> = props => {
         )
 
         if (isEigen) {
-          if (order.mode === "OFFER" && order.source === "artwork_page") {
-            window?.ReactNativeWebView?.postMessage(
-              JSON.stringify({
-                key: "goToInboxOnMakeOfferSubmission",
-                orderCode: order.code,
-                message: `The seller will respond to your offer by ${order.stateExpiresAt}. Keep in mind making an offer doesn’t guarantee you the work.`,
-              })
-            )
-            // We cannot expect Eigen to respond all the time to messages sent from the webview
-            // a default fallback page is safer for old/broken Eigen versions
-            setTimeout(() => {
-              router.push(`/orders/${order.internalID}/status`)
-            }, 500)
-            return
+          if (order.mode === "OFFER") {
+            if (order.state === "IN_REVIEW") {
```

Comment:
Instead of using the order's state, what do you think about checking whether the order has a conversation ID or not? That way, we can continue to avoid using this state in Force. (We can also avoid refactoring this logic when we introduce a buyer status.)

Reply:
OK, I see the limitation of expecting a conversation ID to be available. I also think that we're limited by the fact that Exchange does not report this state back to users. We may need to check the 3 things here, like we have to do in Metaphysics: mode is offer, payment is wire, state is submitted.

Reply:
good point - done in [acc19f3](https://github.com/artsy/force/pull/11914/commits/acc19f3e2f2c167f414c0ca51ad1ee0cfb6c97a5)